modules/articles/tests/client/articles.client.controller.tests.js

```diff
@@ -1,14 +1,14 @@
-'use strict';
-
 (function () {
```

Comment:
Its already this way in master.  You want me to remove the IIFEs from the tests as well?

https://github.com/meanjs/mean/blob/master/modules/articles/tests/client/articles.client.controller.tests.js